.github/workflows/auto_tag.yaml

```diff
@@ -0,0 +1,30 @@
+name: Essential files modified
+
+on:
+  push:
+    branches:
+    - master
+    paths-ignore:
```

Comment:
What do you think about explicitly including paths/files we consider essential instead of including these dirs?

Reply:
Good point, i will align to the labeler config.